README.md

```diff
@@ -102,7 +102,7 @@ tests locally as well.
 # References
 
 * [Primary mkosi git repository on GitHub](https://github.com/systemd/mkosi/)
-* [mkosi — A Tool for Generating OS Images](http://0pointer.net/blog/mkosi-a-tool-for-generating-os-images.html) introductory blog post by Lennart Poettering (2017)
+* [A re-introduction to mkosi -- A Tool for Generating OS Images](https://0pointer.net/blog/a-re-introduction-to-mkosi-a-tool-for-generating-os-images.html)
```

Comment:
Can we make this a real en dash (Compose key + `--.` = –
```suggestion
* [A re-introduction to mkosi – A Tool for Generating OS Images](https://0pointer.net/blog/a-re-introduction-to-mkosi-a-tool-for-generating-os-images.html)
```
or an em dash (Compose key + `---` = —)
```suggestion
* [A re-introduction to mkosi—A Tool for Generating OS Images](https://0pointer.net/blog/a-re-introduction-to-mkosi-a-tool-for-generating-os-images.html)
```
Github renders this a bit ugly as two hyphens, which results in a broader dash with a light region in the middle. The same happens too in the blog post, I just noticed.